action/alarmWebAction.js

```diff
@@ -191,6 +191,11 @@ function main(params) {
                         reason: doc.status.reason
                     }
                 };
+                var strict = true; // strict is default to true
+                if (doc.strict !== undefined) {
+                    strict = doc.strict;
+                }
+                body.config.strict = strict;
```

Comment:
Do we need any updates in documentations?

Reply:
I think document added by #198 is enough